etl/pypasar/db/sql/postgres/drug_exposure/temp_drugmed_view.sql

```diff
@@ -0,0 +1,232 @@
+-- ***********************************************************************
+-- NAME: temp_drug_med_view.sql
+-- Create temp view that joins intraop.drugmed with source_to_concept_map
+-- ***********************************************************************
+
+
+create or replace view {OMOP_SCHEMA}.{DRUGMED_STCM_VIEW} as
+with 
+dm as (
+	select 
+		id,
+		anon_case_no_clindoc,
+		anon_case_no,
+		session_id,
+		operation_id,
+		medication_startdate as "drug_exposure_start_date",
+		medication_startdatetime as "drug_exposure_start_datetime",
+		operation_enddate as "drug_exposure_end_date",
+		(operation_enddate::text || ' ' || operation_endtime)::timestamp as "drug_exposure_end_datetime",
+		dosage_individual as "quantity",
```

Comment:
Documented this assumption since this is taken as is without drug_strength table